api/src/main/java/org/apache/cloudstack/api/BaseCmd.java

```diff
@@ -264,6 +269,10 @@ public String getActualCommandName() {
      */
     public abstract long getEntityOwnerId();
 
+    public List<Long> getEntityOwnerIds() {
+        return null;
```

Comment:
Could this cause any exception; maybe return empty array instead or null?

Reply:
the only place referenced - ParamProcessWorker class has been handled to check if a null is returned